src/constants/line/data-helpers-line.js

```diff
@@ -4,7 +4,7 @@ export function generateSeriesForPureRandom() {
   for (let i = 0; i < numberOfSeries; i++) {
     let currentData = [];
     const currentName = `Random Serie ${i + 1}`
-    const min = Math.random() * 1000 - 500;
```

Comment:
This one is to avoid incorrect representation of areas in the case of negative numbers in series

Reply:
Good job! It makes sense! :)